frontend/src/utils/tsxUtils.tsx

```diff
@@ -545,3 +562,12 @@ export function IconButton(p: { onClick?: React.MouseEventHandler<HTMLElement>;
         </Tooltip>
     );
 }
+
+
+export const navigatorClipboardErrorHandler = (e: DOMException) => {
```

Comment:
We may want to check which browser support this https://caniuse.com/?search=writeText

Reply:
![Screenshot 2023-09-25 at 14 11 01](https://github.com/redpanda-data/console/assets/1083817/d599ecff-7f0d-47a3-906a-61f8f8d5850b)

I think what is problematic is `GPUQueue API: writeTexture`, we're using `Clipboard API: writeText`